src/components/forms/constructor-form/constructor-form.js

```diff
@@ -58,23 +66,26 @@ const ConstructorForm = ({ isEdit, editableConstructorElement }) => {
   const history = createBrowserHistory();
 
   const {
-    constructorImg,
-    setConstructorImg,
-    createConstructor
+    createConstructor,
+    setUploadConstructorImg,
+    uploadConstructorImg
   } = useConstructorHandlers();
 
-  const {
-    list,
-    model,
-    constructorElementMethod,
-  } = useSelector(selectConstructorMethodAndMaterials);
+  const { list, model, constructorElementMethod } = useSelector(
+    selectConstructorMethodAndMaterials
+  );
 
-  const [materialColors, setMaterialColors] = useState([])
+  const [materialColors, setMaterialColors] = useState([]);
+  const [constructorAvatar, setConstructorAvatar] = useState('');
 
   useEffect(() => {
     if (isEdit) {
-      setConstructorImg(editableConstructorElement.image);
-      setMaterialColors(list.filter(el=>el._id===editableConstructorElement.material._id)[0].colors);
+      setConstructorAvatar(`${IMG_URL}${editableConstructorElement.image}`);
```

Comment:
You don't give possibility to admin to see avatar of photo that he added using input. Are you sure?)

Reply:
![image](https://user-images.githubusercontent.com/46137635/107725056-f9448b80-6ced-11eb-8a2c-c572edce13a4.png)